playbooks/tests/tasks/lbaas.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I think this is missing some basic lbaas v2 tests. Example, creating pool, listener, monitors, adding members, etc.  Please reach out to Sangeetha to get test coverage improved for lbaas.